Wobble/Graphics/Sprites/AnimatableSprite.cs

```diff
@@ -22,6 +22,16 @@ public class AnimatableSprite : Sprite
         /// </summary>
         public int CurrentFrame { get; private set; }
 
+        /// <summary>
+        ///     The frame the animation start from.
+        /// </summary>
+        public int DefaultFrame { get; set; }
```

Comment:
DefaultFrame -> StartFrame? 

Not sure why it's called default.

Reply:
Dunno why for some reason default came first in my head, but yeah Start work better, changed.

Also added Rows and Columns value, which can be used to keep in memory those value inside the sprite when loading a spritesheet, in case those are needed when row/column represent different thing (ex: hitobjectspritsheet)